runtime/connectors/gcs/gcs.go

```diff
@@ -70,3 +78,44 @@ type connector struct{}
 func (c connector) Spec() connectors.Spec {
 	return spec
 }
+
+// Consume will eventually be added to the interface
+func Consume(ctx context.Context, source *connectors.Source) (string, error) {
+	conf, _ := ParseConfig(source.Properties)
```

Comment:
Probably handle and return the config error – maybe we add validation there in future